lib/topic_view.rb

```diff
@@ -820,7 +820,7 @@ def filter_posts_by_ids(post_ids)
   def find_topic(topic_or_topic_id)
     return topic_or_topic_id if topic_or_topic_id.is_a?(Topic)
     # with_deleted covered in #check_and_raise_exceptions
-    Topic.with_deleted.includes(:category, :tags).find_by(id: topic_or_topic_id)
+    Topic.with_deleted.includes(:category).find_by(id: topic_or_topic_id)
```

Comment:
This will actually perform one less query as `tags` is an association going through `topic_tags` and AR is issuing a join instead of two queries.

Reply:
Any chances that this was added because a plugin was generating N+1? 🤔

Reply:
No, actually I’ve added it the other day when checking for N+1 when you raised the issue in another PR :sweat_smile: I just scanned through the code, saw that we were including `category` but not `tags`, so I just added `tags` here.